### DISCUSSION TITLE
Mock image/S3 forbidden access

### DISCUSSION THREAD

**Discussion by fabiobarboza7** [mockoon/mockoon#373] [Help] [answered]
Right after the last uptdate to mockmoon 1.11.0 mock images are getting aa 403 HTTP code as can be seen below:

![Captura de Tela 2020-11-19 às 21 17 45](https://user-images.githubusercontent.com/39923194/99740233-51803e80-2aad-11eb-99b4-76681e2fba18.png)

**Comment by 255kb** [comment, marked as answer]
Unfortunately, things changed for the library we are using, Faker.js. The maintainer decided not working for free anymore, so probably cut access to the S3: https://github.com/Marak/faker.js/issues/1054
The fact that this occurred after update 1.11.0 is just a coincidence. 

I need to assess whether we should use another library, or reimplement the problematic helpers (namely images). 
I created a bug issue in the mockoon/commons#7 repository to follow this.
